api/trace_service.proto

```diff
@@ -0,0 +1,47 @@
+syntax = "proto3";
+
+// [#proto-status: draft]
+
+package envoy.api.v2;
+
+import "api/base.proto";
+import "api/grpc_service.proto";
+import "trace.proto";
+
+import "google/api/annotations.proto";
+
+import "validate/validate.proto";
+
+// Service for streaming traces to server that consumes the trace data. It
```

Comment:
This sentence can be simplified: An interface for streaming traces from a client to a server.

There is no need to mention "consumes the trace data". The server can just forward the data elsewhere. There is no need to speculate the usage.